gcloud-java-core/src/main/java/com/google/gcloud/BaseServiceException.java

```diff
@@ -138,7 +146,7 @@ public BaseServiceException(int code, String message, String reason, boolean ide
     this.code = code;
     this.reason = reason;
     this.idempotent = idempotent;
-    this.retryable = idempotent && new Error(code, reason).isRetryable(retryableErrors());
+    this.retryable = new Error(code, reason).isRetryable(idempotent, retryableErrors());
```

Comment:
I think this code could be cleaned a little. I sent @mderka a small patch to consider.